go.mod

```diff
@@ -9,17 +9,17 @@ require (
 	github.com/google/go-cmp v0.3.1
 	github.com/hashicorp/aws-sdk-go-base v0.3.0
 	github.com/hashicorp/go-version v1.2.0
-	github.com/hashicorp/hcl v0.0.0-20180404174102-ef8a98b0bbce // indirect
+	github.com/hashicorp/hcl/v2 v2.0.0
 	github.com/hashicorp/hcl2 v0.0.0-20190821123243-0c888d1241f6
 	github.com/hashicorp/logutils v1.0.0
-	github.com/hashicorp/terraform v0.12.9
+	github.com/hashicorp/terraform v0.12.10-0.20191002221021-39e609d5fd70
```

Comment:
Is there a way to keep this version as 0.12.9?

Reply:
https://github.com/hashicorp/terraform/commit/39e609d5fd708a3e2c63b347b2890e93b79e06af They switched to the new repo in this commit and some types break without these changes.